tests/plugins/chart/basic_chart.test.ts

```diff
@@ -1588,52 +1593,113 @@ describe("Chart design configuration", () => {
-        const data = runtime.chartJsConfig.data!;
-        const tooltipItem = getTooltipItem(data.datasets![0].data![0] as number, chartType);
-        expect(
-          // @ts-ignore should be binded to the chart tooltip model
-          runtime.chartJsConfig.options!.plugins!.tooltip!.callbacks!.label!(tooltipItem)
-        ).toEqual("$6,000.00");
-      }
-    );
+    test.each(["bar", "line"])("Basic chart tooltip label, cell with format", (chartType) => {
+      setCellContent(model, "A2", "6000");
+      setCellFormat(model, "A2", "[$$]#,##0.00");
+      createChart(model, { ...defaultChart, type: chartType as "bar" | "line" | "pie" }, "42");
+      const runtime = model.getters.getChartRuntime("42") as BarChartRuntime;
+      const data = runtime.chartJsConfig.data!;
+      const tooltipItem = getTooltipItem(
+        data.datasets![0].data![0] as number,
+        chartType,
+        runtime.chartJsConfig
+      );
+      expect(
+        // @ts-ignore should be binded to the chart tooltip model
+        runtime.chartJsConfig.options!.plugins!.tooltip!.callbacks!.label!(tooltipItem)
+      ).toEqual("$6,000.00");
+    });
 
-    test.each(["bar", "line", "pie"])(
-      "Basic chart tooltip label, date format is ignored",
-      (chartType) => {
-        setCellContent(model, "A2", "6000");
-        setCellFormat(model, "A2", "m/d/yyyy");
-        createChart(model, { ...defaultChart, type: chartType as "bar" | "line" | "pie" }, "42");
-        const runtime = model.getters.getChartRuntime("42") as BarChartRuntime;
-        const data = runtime.chartJsConfig.data!;
-        const tooltipItem = getTooltipItem(data.datasets![0].data![0] as number, chartType);
-        expect(
-          // @ts-ignore should be binded to the chart tooltip model
-          runtime.chartJsConfig.options!.plugins!.tooltip!.callbacks!.label!(tooltipItem)
-        ).toEqual("6,000");
-      }
-    );
+    test.each(["bar", "line"])("Basic chart tooltip label, date format is ignored", (chartType) => {
+      setCellContent(model, "A2", "6000");
+      setCellFormat(model, "A2", "m/d/yyyy");
+      createChart(model, { ...defaultChart, type: chartType as "bar" | "line" | "pie" }, "42");
+      const runtime = model.getters.getChartRuntime("42") as BarChartRuntime;
+      const data = runtime.chartJsConfig.data!;
+      const tooltipItem = getTooltipItem(
+        data.datasets![0].data![0] as number,
+        chartType,
+        runtime.chartJsConfig
+      );
+      expect(
+        // @ts-ignore should be binded to the chart tooltip model
+        runtime.chartJsConfig.options!.plugins!.tooltip!.callbacks!.label!(tooltipItem)
+      ).toEqual("6,000");
+    });
+  });
+  describe("Pie Chart tooltip", () => {
+    test("pie chart tooltip label to include percentage", () => {
+      setCellContent(model, "A1", "P1");
+      setCellContent(model, "A2", "60000000");
+      createChart(
+        model,
+        {
+          dataSets: ["A1:A2"],
```

Comment:
I'd add a cell with another number value in the dataset, to check that the percentage is correclty computed. (no need to test both tooltips)